scripts/lighthouse/fixtures.js

```diff
@@ -0,0 +1,67 @@
+const url = 'http://localhost:7080/news/articles/c9rpqy7pmypo';
```

Comment:
Having looked through the PR, not quite sure of the relationship between this file and the top-level `lighthouse.js` file. Can that be documented somewhere?

Reply:
I'd suggest this as a follow up issue

Reply:
This reminds me I should add a README describing the purpose of `simorgh/ligthouse.js`so I have done that. The fixture data are specific to the tests. L1 could be any URL.